src/PointwiseFunctions/GeneralRelativity/DerivativeSpatialChristoffel.hpp

```diff
@@ -0,0 +1,50 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "DataStructures/Tensor/TypeAliases.hpp"
+#include "Utilities/Gsl.hpp"
+
+namespace gr {
+/// @{
+/*!
```

Comment:
Because of the symmetrization in of the second derivative I'm less sure this is not CCZ4 specific. Thinking about this more, I think it's better to code up the exact equation in the paper and move to the CCZ4 directory. That would also fix the 1/2 factor missing below

Reply:
I had the exact same thought yesterday, but just figured I was thinking that because I was doing the math wrong. I'll have a go at your above suggestion with testing against a numerical derivative.

Reply:
After looking at the regular equation for the christoffel symbols of the second kind, I agree that the generic (non-conformal) derivative isn't usable for CCZ4 due to the symmetrization. I can move this work, along with the requested fixes, into the `Ccz4` directory.

Do you have a preference of me closing this PR and opening a new one vs. fixing this one? The only real reason I ask is nitpicky, which is that the branch name `gr-deriv-spatial-christoffel` no longer matches the goal (it won't be a generic `gr` function), and I didn't know if that mattered for any reason.

Reply:
It's up to you. Nobody other than you sees the branch names really. I often name mine things like `ghgrmhd_subcell_0` which is also not very descriptive :)